client/my-sites/earn/memberships/index.jsx

```diff
@@ -497,19 +498,7 @@ class MembershipsSection extends Component {
 	renderStripeConnected() {
 		return (
 			<div>
-				{ this.props?.query?.stripe_connect_success === 'earn' && (
-					<Notice
-						status="is-success"
-						showDismiss={ false }
-						text={ this.props.translate(
-							'Congrats! Your site is now connected to Stripe. You can now add your first payment plan.'
-						) }
-					>
-						<NoticeAction href={ `/earn/payments-plans/${ this.props.siteSlug }` } icon="create">
-							{ this.props.translate( 'Add a payment plan' ) }
-						</NoticeAction>
-					</Notice>
-				) }
+				{ this.renderNotices() }
```

Comment:
Nice move!

Reply:
> The idea is great but from my understanding it is breaking the "original" flo by not displaying the 'Congrats! Your site is now connected to Stripe. You can now add your first payment plan.' notice...

Great question, the original message will be displayed when the `'#add-newsletter-payment-plan`  hash is not in the URL as we are determining the source to send to the backend based on that hash.